include/bgfx/bgfx.h

```diff
@@ -645,6 +645,8 @@ namespace bgfx
 		                                   ///  create back-buffer color surface.
 		void* backBufferDS;                //!< Backbuffer depth/stencil. If `NULL`, bgfx will create a back-buffer
 		                                   ///  depth/stencil surface.
+		void *worldTracking;               //!< Native world tracking provider used for visionOS.
```

Comment:
> Another possible solution is to run the ARSession inside BGFX, but this may limit the user's ability to run other providers like Hand Tracking Provider, etc.

I would prefer this as first implementation, since it doesn't require changing API.